lib/CGenParser_lib.ml

```diff
@@ -193,7 +193,7 @@ module Do
         filter = filter;
         condition = final;
         locations = locs;
-        extra_data = MiscParser.CExtra params;
+        extra_data = [MiscParser.CExtra params; (snd prog_litmus)];
```

Comment:
If I am not wrong, `(snd prog)` (or `data`) does not appear anywhere. This question the type of `deep_parser`.